modules/customize-certificates-understanding-service-serving.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="understanding-service-serving-{context}"]
+= Understanding service serving certificate secrets
+
+Service serving certificate secrets are intended to support complex 
+middleware applications that need out-of-the-box certificates. It has 
+the same settings as the server certificates generated by the 
+administrator tooling for nodes and masters.
+
+Once the cluster generates a secret for your service, your PodSpec can
+mount it and the Pod will run as it becomes available. The generated
+certificate will be valid for the internal service DNS name 
+`<service.name>.<service.namespace>.svc`.
+
+The generated certificate and key are in PEM format, stored in `tls.crt` 
+and `tls.key` respectively, within the created secret. The 
+certificate and key pair is automatically replaced when it gets close to 
+expiration. 
+
+The signature algorithm in use is `x509.SHA256WithRSA`.
```

Comment:
I'd move this line earlier in the procedure so the bit about expiration is closer to the note. I'd also make some mention of the expiration period in the concept in case it gets reused without the note.

Reply:
I've reworked this content using the suggestions above. The expiration is now listed in the module, and the note only refers to the automatic upgrade and manual rotation.